### ISSUE TITLE
Labs should support placeholder objects

### ISSUE THREAD

**Issue by leeyh20** [PowerPointLabs/PowerPointLabs#1599]
After #1554  is settled, we can use the same functions in ShapeUtil to allow other labs to support placeholders. 

1. Effects Lab #1536 
2. Shapes Lab #1591 
3. ConvertToPicture 

This is the parent issue for the above issues.

**Comment by YuPeiHenry** [comment]
Additionally, I observed that currently support for placeholders are **inconsistent** in `Shapes Lab`. It is unsupported on most slides. However, in `PPT2016`, if you add a blank slide as shown below, any empty placeholder objects can be added to `Shapes Lab`.
![image](https://user-images.githubusercontent.com/40821389/51546573-6682eb00-1e9f-11e9-8573-26689c890baf.png)

This is caused by the conversion of the placeholder object to a `MsoAutoShape`. I wish to address this in a separate PR.

**Comment by yuhongherald** [comment]
The PR #1915 uses an existing fix that allows pasting of placeholders as lookalike objects, as a workaround of a specific number of placeholders on the same slide.
